src/main/java/uk/gov/hmcts/reform/profilesync/util/JsonFeignResponseUtil.java

```diff
@@ -22,15 +22,17 @@
 @SuppressWarnings("unchecked")
 @Slf4j
 public class JsonFeignResponseUtil {
-    private static final ObjectMapper json = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    private static final ObjectMapper json = new ObjectMapper().configure(
+            DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
_(Non-essential)_ Could you move `.configure(` onto the next line also so it reads better?

Reply:
Done.